public/static/pathways/her2_pathway.json

```diff
@@ -68,17 +68,8 @@
           "type": "create",
           "description": "Chemotherapy TH Weekly",
           "resource": {
-            "resourceType": "ServiceRequest",
-            "code": {
-              "coding": [
-                {
-                  "system": "http://snomed.info/sct",
-                  "code": "367336001",
-                  "display": "Chemotherapy (procedure)"
-                }
-              ],
-              "text": "Chemotherapy (procedure)"
-            }
+            "resourceType": "CarePlan",
+            "title": "ChemotherapyTH"
```

Comment:
So it looks like we are using the title for the CQL but should we update to actually include the correct information for the medication requests?

Reply:
From the sprint review meeting yesterday it sounds like this was the goal of the task so no need to update anything

Reply:
Yeah we have a task in the backlog to fill out more details on the CarePlan, but that's going to involve more UI work so I think that's going to be on the back burner for a while.